apps/settings/js/panels/addons/addons.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Handle addons panel's functionality.
+ *
+ * @module AddonsManager
+ */
+define(function(require) {
+  'use strict';
+
+  var AppsCache = require('modules/apps_cache');
+  var Observable = require('modules/mvvm/observable');
+
+  var AddonsManager = function am() {
+    this.addons = Observable({
```

Comment:
Why not simply `this.addons`? And it should be an `ObervableArray` for `ListView` to reflect the change correctly.